src/app/deleted/deleted.component.ts

```diff
@@ -9,7 +10,7 @@ import { UserStructure } from '../user-structure.model';
 })
 export class DeletedComponent implements OnInit {
 
-  deletedUsers: UserStructure[];
+  deletedUsers: Observable<UserStructure[]>;
```

Comment:
See my comment for naming observables above and also change the model name to `User`